README.md

```diff
@@ -129,6 +145,18 @@ This would prevent Flowzone from finding and finalizing existing draft artifacts
 
 You can read more about the available merge methods [here](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/about-merge-methods-on-github).
 
+### External Contributions
+
+Flowzone supports external contributions (ie. PRs from forks) to your repository by using [`pull_request_target`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) events in addition to `pull_request`. This allows access to secrets normally available to members of the organization or the repository.
+
+To mitigate the risk of secrets being exposed by malicious pull requests, we have taken the following steps in the Flowzone workflow.
+
+- Trusted code includes the Flowzone workflow itself and cannot be modified by pull requests.
+- Untrusted or arbitrary code can be called by Flowzone (eg. npm scripts) but does not expose any secrets in the environment at these points.
+- Custom actions are disabled for external contributors by default and can be [enabled](#restrict_custom_actions) after the custom action has been vetted to not leak secrets to untrusted code.
+
+See the [usage examples](#usage) to get started and allow external contributions to your repo.
+
```

Comment:
I was thinking we would link to this section of the README from the All-hands doc next Mon.

Reply:
Can't hurt.